src-java/nbworker-topology/nbworker-storm-topology/src/main/java/org/openkilda/wfm/topology/nbworker/services/FlowOperationsService.java

```diff
@@ -342,13 +344,16 @@ private List<PathNodePayload> buildPathFromFlow(Flow flow, FlowPath flowPath) {
     /**
      * Partial update flow.
      */
-    public Flow updateFlow(FlowOperationsCarrier carrier, FlowPatch flowPatch) throws FlowNotFoundException {
+    public Flow updateFlow(FlowOperationsCarrier carrier, FlowPatch flowPatch)
+            throws FlowNotFoundException, InvalidFlowException {
         String flowId = flowPatch.getFlowId();
         if (yFlowRepository.isSubFlow(flowId)) {
             throw new MessageException(ErrorType.REQUEST_INVALID, "Could not modify flow",
                     format("%s is a sub-flow of a y-flow. Operations on sub-flows are forbidden.", flowId));
         }
 
+        ValidatorUtils.validateMaxLatencyAndLatencyTier(flowPatch.getMaxLatency(), flowPatch.getMaxLatencyTier2());
+
```

Comment:
This is not correct way to check latency in patch. 

Patch request can contain only fields which must be changed. For example patch request can have body 
```
{
    max_latecny_tire2: 123
}
```
Without max_latency.
If some field is missed - we must take field value from already existent flow. 
(look at lines 370-380) for example. Only after that we can compare latencies

Reply:
fixed